airflow/utils/log/file_task_handler.py

```diff
@@ -132,6 +132,19 @@ def _render_filename(self, ti: TaskInstance, try_number: int) -> str:
     def _read_grouped_logs(self):
         return False
 
+    @staticmethod
+    def _try_read_from_k8s(queue):
```

Comment:
This does not actually read from k8s, should it be renames to, say, `should_read_from_k8s`?

Reply:
yeah it's like "_should maybe_ try read from k8s"

Reply:
ok @uranusjr ready for another look... and added test